### DISCUSSION TITLE
refine update with bun

### DISCUSSION THREAD

**Discussion by wguerram** [refinedev/refine#6294] [General]
Hi, when I run command

`
bun run refine update
`

I would expect refine to use bun install instead of yarn add for updating the components. How can I force bun install to be run?

Does refine update does additional updates in the configurations or just updates the packages? Is it safe to run bun install for every refine component to upgrade version for example 

`
bun install @refinedev/cli
`

Or the recommendation is to rely on refine update?

**Comment by BatuhanW** [comment]
Hey @wguerram, we didn't test Refine with bun and currently our CLI doesn't support it. Refine update should still update your package versions in package.json, so you can use bun install after that.
